### PR TITLE
Lint influx_tsm; Relates to #4098

### PR DESCRIPTION
**Pull request by sdtsui**
###### Required for all non-trivial PRs
- [x] Rebased/mergable
- [x] Tests pass
- [x] CHANGELOG.md updated
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

###### Required only if applicable
_You can erase any checkboxes below this note if they are not applicable to your Pull Request._

- Single commit, PR follows conventions laid out by @Gouthamve in #5822
* main.go: struct field CpuFile should be CPUFile
* influx_inspect: loop equivalent to `for key := range...`
* adds comments to exported fields and consts
* updates typo in 529 of `CHANGELOG.md`: text for #4702 now matches number

### CODE REVIEW

**Comment by sdtsui** [comment]
Refs #4098 .

**Comment by e-dard** [comment]
@sdtsui thanks! This looks good. I just need to check one thing with the team and then I'll merge this.

File: cmd/influx_tsm/stats/stats.go [hunk 1]
```diff
@@ -18,30 +18,37 @@ type Stats struct {
 	TotalTime       time.Duration
 }
 
+// AddPointsRead adds read points
```
Comment (jsternberg) [inline cmd/influx_tsm/stats/stats.go]: Can you make this docstring more useful with something like:

```
// AddPointsRead increments the number of read points.
```

Same with the other docstrings below this. They should also have periods at the end of each sentence.

Comment (sdtsui) [inline cmd/influx_tsm/stats/stats.go]: Thank you, @jsternberg, for the example.  Let me know if you have other suggestions. I can do more on the weekend and will follow the same format/guidelines.

**Comment by sdtsui** [comment]
Updated with better comments. Let me know if I can do anything else to help this get merged. Thanks!

**Comment by jsternberg** [comment]
👍

**Pull request merged**